app/javascript/dashboard/routes/dashboard/notifications/components/NotificationPanelList.vue

```diff
@@ -0,0 +1,211 @@
+<template>
+  <div class="notification-list-item--wrap h-full flex-view ">
+    <div
+      v-for="notificationItem in notifications"
+      v-show="!isLoading"
+      :key="notificationItem.id"
+      class="notification-list--wrap flex-view "
+      @click="() => onClickNotification(notificationItem)"
+    >
+      <div
+        v-if="!notificationItem.read_at"
+        class="notification-unread--indicator"
+      ></div>
+      <div v-else class="empty flex-view"></div>
+      <div class="notification-content--wrap w-full flex-space-between">
+        <div class="flex-space-between">
+          <div class="title-wrap flex-view ">
+            <span class="notification-title">
+              {{
+                `#${
+                  notificationItem.primary_actor
+                    ? notificationItem.primary_actor.id
+                    : 'deleted'
```

Comment:
i18n label for "deleted"

Reply:
fixes by https://github.com/chatwoot/chatwoot/pull/4061/commits/7b9a882462717146eb4a017b06820631b6e93003